app/com/linkedin/drelephant/util/Utils.java

```diff
@@ -234,11 +234,12 @@ public static int getHeuristicScore(Severity severity, int tasks) {
    *
    * @param field the field to br truncated
    * @param limit the truncation limit
+   * @param context additional context for logging purposes
```

Comment:
I was wondering why `truncateField` was taking an `appId`, then I saw what was going on here. Hopefully this change helps.